### ISSUE TITLE
Logging training set metrics

### ISSUE THREAD

**Issue by karl-joan** [ultralytics/yolov5#10868]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hey! Is there an easy way to log metrics (that are already logged for the validation set) for the training set?

**Comment by glenn-jocher** [comment]
@karl-joan hey there! Sure, you can easily log training set metrics by adding the `--log-imgs` flag to `train.py`. This will log training set images and metrics every `log_interval` iterations as defined in your `data.yaml`. The metrics include loss, average precision, and so on. Check out our documentation for more details on training and logging options. Let me know if you need any further assistance!
